db-service/lib/deep-genres.sql

```diff
@@ -0,0 +1,126 @@
+-- DEBUG => this.dbc._native.prepare(cds.utils.fs.readFileSync(__dirname + '/deep-genres.sql','utf-8')).exec([JSON.stringify(query.UPDATE.data)])
+DO (IN JSON NCLOB => ?) BEGIN
+
+  -- Extract genres with a depth of 3 (like: '$.children[*].children[*]')
```

Comment:
depth can be determined based on input json, right? we must just `union all` all provided rows